packages/devtools/lib/src/timeline/cpu_profile_protocol.dart

```diff
@@ -39,6 +39,11 @@ class CpuProfileData {
 
   void _processStackFrames(Response response) {
     final nativeRoot = CpuStackFrame('nativeRoot', '[Native]', 'Dart');
+    final nativeTruncatedRoot = CpuStackFrame(
+      'nativeTruncatedRoot',
```

Comment:
nit: looking at this code makes me think CpuStackFrame should take required named arguments. That would make it easier for me to tell what these 3 string arguments are when reviewing the code.

Reply:
done.